src/free-trial/tax/components/partner-card.scss

```diff
@@ -0,0 +1,50 @@
+@import '../../../stylesheets/variables';
```

Comment:
I'd prefer to import this as a global import rather than each required file. I thought of using Webpack but wasn't sure how to do it proper. Any thoughts?

Reply:
I 100% agree! I think we can set up a sass-loader `additionalData` like [this](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/internal-style-build/index.js#L48-L55).

Reply:
Thanks, @chihsuan! I was trying this out but it didn't turn out too great as I keep on importing new packages. I'm thinking to do this in a follow-up 🙏

Reply:
Yeah, sounds great!